test/js-native-api/test_string/test.js

```diff
@@ -130,6 +130,19 @@ assert.strictEqual(test_string.TestPropertyKeyUtf16AutoLength(str6), str6);
 assert.strictEqual(test_string.Utf16Length(str6), 5);
 assert.strictEqual(test_string.Utf8Length(str6), 14);
 
+const str7 = 'propertyKey';
+assert.strictEqual(test_string.TestPropertyKeyUtf8(str7), str7);
+
+const obj1 = test_string.TestSetNamedPropertyLen();
+assert.strictEqual(typeof obj1, 'object');
+assert.strictEqual(Object.keys(obj1).length, 1);
+assert.strictEqual(obj1['\0test'], 42);
+
+const obj2 = test_string.TestSetNamedPropertyLenAutoLength();
```

Comment:
The tests for the TestSetNamedPropertyLen() and TestSetNamedPropertyLenAutoLength() must be removed as they are not exposed by the `test_string.c` anymore.
Instead, please try to follow the same test pattern as in the PR #50282